deps/parser/parser.jl

```diff
@@ -30,26 +30,39 @@ type_translate_list = Dict(
 
 ## Dummy, will soon be generated
 filenames_list = Array{String,1}(
-    [ abspath(joinpath("tmp","polytope.json")) ]
+    [ abspath(joinpath( @__DIR__, "..", "..", "tmp", "polytope.json")) ]
 )
 
 function julia_function_string(julia_name::String, polymake_name::String, app_name::String, julia_args::String, call_args::String, parameter_string::String )
+
     return """
-function $julia_name( $julia_args )
+function $julia_name( $julia_args, ::Val{false} )
     $(typestring)application( "$app_name" )
     return $(typestring)call_function( "$polymake_name$parameter_string", Array{Any,1}([ $call_args ]) )
 end
 
+function $julia_name( $julia_args )
+    $(typestring)application( "$app_name" )
+    return_value = $(typestring)call_function( "$polymake_name$parameter_string", Array{Any,1}([ $call_args ]) )
+    return convert_from_property_value(return_value)
+end
```

Comment:
You could also have just one function definition, the compiler figures that out.

```julia
function $julia_name( $julia_args , ::Val{Convert}=Val(true))
     $(typestring)application( "$app_name" )
     return_value = $(typestring)call_function( "$polymake_name$parameter_string", Array{Any,1}([ $call_args ]) )
     if Convert
         return convert_from_property_value(return_value)
    else
        return return_value
    end
end
```

Reply:
I see this works. I will do this.